invenio_communities/administration/communities.py

```diff
@@ -96,7 +96,7 @@ def disabled():
 class CommunityDetailView(AdminResourceDetailView):
     """Admin community detail view."""
 
-    url = "/communities/<pid_value>"
+    url = "/communities/<pid_value>/records"
```

Comment:
question: why the url was changed to `.../records`? This affects the URL that is auto generated. I.e. in Zenodo this generates the views https://zenodo.org/administration/communities/<uuid>